_data-prepper/pipelines/configuration/processors/add-entries.md

```diff
@@ -1,25 +1,62 @@
 ---
 layout: default
-title: add_entries
+title: Add entries processor
```

Comment:
I suggested in #3666 that we use the plugin name here in the title to help with consistency.

Reply:
I'll address that in a separate PR if we decide to change it. When we discussed this as a docs team, we thought that keeping the title underscore free would be better for SEO. Open to further discussion on that though.